doc/howto/network_bgp.md

```diff
@@ -4,20 +4,60 @@ discourse: 11567
 
 # How to configure LXD as a BGP server
 
-LXD can act as a BGP server, effectively allowing to establish sessions with upstream BGP routers and announce the addresses and subnets that it's using.
+```{note}
+The BGP server feature is available for the {ref}`network-bridge`, the {ref}`network-physical` and the {ref}`bridged NIC type <instance_device_type_nic_bridged>`.
```

Comment:
So we basically have:
 - Bridge networks with BGP
   - ipv4.address subnet is advertised if ipv4.nat isn't true
   - ipv6.address subnet is advertised if ipv6.nat isn't true
   - Network forward addresses are advertised
   - Any address or subnet specified in ipv4.routes.external or ipv6.routes.external on an instance connected to the bridge network will be advertised
 - Physical networks with BGP
   - Nothing is done directly at the level of that network
   - All downstream networks (those that specify the physical network as their network= during creation) will have their networks, forwards and routes advertised similarly to the bridged network case above

Reply:
We should mention OVN networks here also?

Reply:
> We should mention OVN networks here also?

OVN networks would be downstream, right? The idea here is to mention on which networks you can enable the feature, and this would not include OVN if I understand correctly.
Discussed with @stgraber that the bridged NIC type should be removed here as well.

Reply:
Ah yes makes sense if ur removing the bridged nic mention from here as that would be downstream too.